### PR TITLE
ClassAttributesSeparationFixer - fix for properties with type alternation

### PR DESCRIPTION
**Pull request by kubawerlos**
Commented part in test can be commented out after https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5550 fix.

### CODE REVIEW

File: tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php [hunk 1]
```diff
@@ -1440,5 +1440,27 @@ public function __construct(
...
             }',
         ];
+
+        yield 'typed properties' => [
+            '<?php
+            class Foo {
+                /*
+                private static int | float | null $a;
+                private static int | float | null $b;
+                */
```
Comment (keradus) [inline tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php:1450]:
```suggestion
                private static int | float | null $a;
                private static int | float | null $b;
```

File: tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php [hunk 2]
```diff
@@ -1440,5 +1440,27 @@ public function __construct(
...
+                private int | float | null $c;
+
+                private int | float | null $d;
+            }',
+            '<?php
+            class Foo {
+                /*
+                private static int | float | null $a;
+                private static int | float | null $b;
+                */
```
Comment (keradus) [inline tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php:1460]:
```suggestion
                private static int | float | null $a;
                private static int | float | null $b;
```

**Review by keradus — APPROVED** [review]
if CI is green

**Comment by kubawerlos** [comment]
Shuold I rebase to make fabbot happy?

**Comment by keradus** [comment]
no, i will squash

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**